README.md

```diff
@@ -26,10 +26,20 @@ html, body {height: 100%; margin: 0;}
 
 body {
   font-family: "Segoe UI", sans-serif;
-  background: #1A2933; color: #FFF;
+  background: #1A2933;
+  color: #FFF;
+}
+
+h1 {
+  margin: 0 0 10px 0;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 10px;
+  color: rgba(255,255,255,0.4);
```

Comment:
I'd rather leave these unimportant styles collapsed in the tutorial

Reply:
Okie!